e2e/tests/e2e_install_che_operatorhub/InstallCheByOperatorHub.spec.ts

```diff
@@ -0,0 +1,119 @@
+/*********************************************************************
+ * Copyright (c) 2019 Red Hat, Inc.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ **********************************************************************/
+
+import { e2eContainer } from '../../inversify.config';
+import { ILoginPage } from '../../pageobjects/login/ILoginPage';
+import { ILoginPageOcp } from '../../pageobjects/openshift/ILoginPageOcp';
+import { CLASSES, TYPES } from '../../inversify.types';
+import { TestConstants } from '../../TestConstants';
+import { Dashboard } from '../../pageobjects/dashboard/Dashboard';
+import { OpenShiftLoginPage } from '../../pageobjects/openshift/OpenShiftLoginPage';
+import { OpenShiftConsole4x } from '../../pageobjects/openshift/OpenShiftConsole4x';
+
+const loginPage: ILoginPage = e2eContainer.get<ILoginPage>(TYPES.LoginPage);
+const ocpLoginPage: ILoginPageOcp = e2eContainer.get<ILoginPageOcp>(TYPES.OcpLoginPage);
+const openShiftLogin: OpenShiftLoginPage = e2eContainer.get(CLASSES.OpenShiftLoginPage);
+const openShiftConsole: OpenShiftConsole4x = e2eContainer.get(CLASSES.OpenShiftConsole4x);
+const dashboard: Dashboard = e2eContainer.get(CLASSES.Dashboard);
+const projectName: string = TestConstants.TS_INSTALL_CHE_PROJECT_NAME;
+const channelName = TestConstants.TS_OCP_UPDATE_CHANNEL_OPERATOR;
+
+suite('E2E', async () => {
+
+    suite('Go to OCP 4.x and wait console OpenShift', async () => {
+        test('Open login page', async () => {
+            await openShiftLogin.openLoginPageOpenShift();
+            await openShiftLogin.waitOpenShiftLoginPage();
+        });
+        test('Log into OCP 4.x', async () => {
```

Comment:
Please add an empty line between "test" blocks for increasing readability.

Reply:
Ok.